tests/test_difference.py

```diff
@@ -0,0 +1,115 @@
+from plothist import make_hist, get_comparison
+import numpy as np
+from pytest import approx, raises
+
+
+def test_difference_weighted_histograms():
+    """
+    Test weighted histograms error.
+    """
+    h1w = make_hist(
+        data=np.random.normal(size=10),
+        bins=10,
+        range=(-5, 5),
+        weights=np.random.normal(size=10),
```

Comment:
If we are reaaaally unlucky, all weights may be ones. You can ignore this comment.